trucks.js

```diff
@@ -1,511 +1,604 @@
-    Facebook: 'https://www.facebook.com/pages/Flair-Taco/197832600252403'
+    Facebook: 'https://www.facebook.com/pages/Flair-Taco/197832600252403',
   },
   {
     name: 'Frank\'s Franks',
-    type: ['Hot Dogs'],
-    schedule: ['Monday', 'Sunday'],
-    payment: ['Cash', 'Cards'],
+    type: ['Hot Dogs',],
+    schedule: ['Monday', 'Sunday',],
+    payment: ['Cash', 'Cards',],
     description: 'Gourmet hot dogs, weiners, sausages, hot links and more, contact at 425.761.9447 or franksfranks3@gmail.com',
-    Facebook: 'https://www.facebook.com/franksfrankshotdogs'
+    Facebook: 'https://www.facebook.com/franksfrankshotdogs',
   },
   {
     name: 'Fruit Chatter Box',
-    type: ['Sweets'],
-    schedule: ['Sunday'],
-    payment: ['Cash', 'Cards'],
+    type: ['Sweets',],
+    schedule: ['Sunday',],
+    payment: ['Cash', 'Cards',],
     description: 'Fruit Chatter Box provides fresh, seasonal fresh fruit in cups or on a stick at local festivals and street fairs.',
     Facebook: 'https://www.facebook.com/fruitchatterbox',
-    Twitter: 'https://twitter.com/fruitchatterbox'
+    Twitter: 'https://twitter.com/fruitchatterbox',
   },
   {
     name: 'Full Tilt Ice Cream',
-    type: ['Sweets'],
-    payment: ['Cash', 'Cards'],
+    type: ['Sweets',],
+    payment: ['Cash', 'Cards',],
     description: 'Handmade ice creams!',
     website: 'http://www.fulltilticecream.com',
     Facebook: 'https://www.facebook.com/fulltilticecreams',
-    Twitter: 'https://twitter.com/fticecream'
-  }
+    Twitter: 'https://twitter.com/fticecream',
+  },
 ];
 
 // this module should support the following methods:
 // getTrucks() - return all trucks
 // getTruck(name) - return the truck object matching 'name'
 // getFoodTypes() - return unique list of all associated food types (underscore has a function to help)
 // filterByDay(day) - return trucks with 'day' in schedule (use your filterByDay function from Module 3 homework)
-// filterByFoodType(foodType) - return trucks with associated 'foodType'
\ No newline at end of file
+// filterByFoodType(foodType) - return trucks with associated 'foodType'
+
+/* eslint-enable max-len */
+/* eslint-disable no-console */
+/* eslint-disable no-undef */
+module.exports = {
+
+  /* eslint-enable no-undef */
+
+  getTrucks: function () {
+    return foodTrucks;
+  },
+
+  getTruck: function (truck) {
+    var foundTruck = 0;
+
+    for (var i = 0; i < foodTrucks.length; i++) {
+      if (foodTrucks[i].name === truck) {
+        var gotTruck = foodTrucks[i];
+        // console.log(foodTrucks[i].name);
+        foundTruck = 1;
+
+        return gotTruck;
+      }
+
+    }
+
+    if (foundTruck === 0) {
+      return (truck + ' truck not found');
+    }
+
+  },
+
+  getFoodTypes: function () {
+    var types;
+    for (var i = 0; i < foodTrucks.length; i++) {
+      types += foodTrucks[i].type + ' -> ' + foodTrucks[i].name + '\n';
```

Comment:
This method should only return the list of types, not a food truck name as well.